cloudformation/tag-manager.yaml

```diff
@@ -303,3 +309,47 @@ Resources:
             Ref: CapiPreviewRole
       Roles:
       - Ref: TagManagerRole
+  Tagmanager5XXAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: IsProd
+    Properties:
+      TreatMissingData: notBreaching
+      AlarmDescription: Tag manager is returning 5x responses
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Threshold: '0'
```

Comment:
I think you might want to raise this threshold slightly ([it's not unusual to see a handful of errors per hour](https://metrics.gutools.co.uk/goto/fd7u7XQ4z?orgId=1)) - perhaps 5 or 10 would be less noisy for the initial threshold?

Reply:
Good idea. Thanks for the review!